rgw/v2/tests/s3_swift/test_sts_using_boto_server_side_copy.py

```diff
@@ -201,10 +202,16 @@ def test_exec(config):
         all_keys_in_buck1 = []
         for obj in bucket1.objects.all():
             all_keys_in_buck1.append(obj.key)
-        copy_source = {"Bucket": bucket1.name, "Key": all_keys_in_buck1[0]}
         copy_object_name = all_keys_in_buck1[0] + "_copied_obj"
         log.info(f"copy object name: {copy_object_name}")
-        bucket2.copy(copy_source, copy_object_name)
+        rgw_s3_client.copy_object(
+            Bucket=bucket2.name,
+            Key=copy_object_name,
+            CopySource={
+                "Bucket": bucket1.name,
+                "Key": all_keys_in_buck1[0],
+            },
+        )
```

Comment:
Currently code is copying only one object from source to destination.
Can we copy all objects from source to destination

Reply:
Sure @udaysk23 . I'll update the code to copy all objects

Reply:
@udaysk23 , I have updated the code to copy all objects. Passed log: http://pastebin.test.redhat.com/1067624